src/components/AppPanel.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div class="panel">
     <header
-      v-if="$slots.heading"
+      v-if="$slots.title"
```

Comment:
Not related to issue. I renamed the slot names. Before was 'header' and 'heading'